### PR TITLE
Feature/milestone five

### PR DESCRIPTION
**Pull request by Maryjanee**
In this milestone, I added functionality to the buttons to enable calculations.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Status: Changes Required :gear:**

**Hello @Maryjanee,**
You have done a good job so far. Your calculator app is working correctly with most of the operations. 

[Required] But I saw that the project breaks when I divide a number by zero.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the requested change and submit the project for another review.**

**If you have any queries regarding the review, please message me on Slack.
Suman Shrestha
Happy coding !!! :slightly_smiling_face:**

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
-const ButtonPanel = () => (
+const Panel = ({ onClick, symbols }) => symbols.map((symbol, index) => (
+  <Button
+    key={(symbol + index).toString()}
+    buttonName={symbol}
+    onClick={onClick}
+  />
+));
+
+const ButtonPanel = ({ onClick }) => (
```
Comment (sumancrest0001) [inline src/components/ButtonPanel.js]: [Required] Hello @Maryjanee, I forgot to mention that you need to create functions in the ButtonPanel and Button component as mentioned in the requirement. Even though your app is running smoothly, it is a hard requirement.  Please reference the [curriculum](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376019/)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Maryjanee  👋

**You've done pretty well fixing issues raised in the previous review.** 👌

However, a new issue cropped up. Just one more step and we are good.

- [ ] After displaying the error from diving by zero and the user clicks a new number, kindly clear the error before displaying the number.
ice_screenshot_20210120-180232

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix this issue and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Review by Torres-ssf — APPROVED** [review]
### Hi @Maryjanee :hand: 
Great job on this milestone :+1: 
Good luck with the next project.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**